radicle-httpd/src/api/v1/projects.rs

```diff
@@ -399,6 +407,89 @@ async fn issues_handler(
     Ok::<_, Error>(Json(issues))
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct IssueCreate {
+    pub title: String,
+    pub description: String,
+    pub tags: Vec<Tag>,
+}
+
+/// Create a new issue
+/// `POST /projects/:project/issues`
+async fn issue_create_handler(
+    State(ctx): State<Context>,
+    AuthBearer(token): AuthBearer,
+    Path(project): Path<Id>,
+    Json(issue): Json<IssueCreate>,
+) -> impl IntoResponse {
+    let sessions = ctx.sessions.write().await;
+    sessions.get(&token).ok_or(Error::Auth("Unauthorized"))?;
+    let storage = &ctx.profile.storage;
+    let signer = ctx.profile.signer().unwrap();
+    let repo = storage.repository(project)?;
+    let mut issues = Issues::open(ctx.profile.public_key, &repo)?;
+    issues
+        .create(issue.title, issue.description, &issue.tags, &signer)
+        .map_err(Error::from)?;
+
+    Ok::<_, Error>(())
+}
```

Comment:
maybe best if you return a `success: true` here instead of empty response?

Reply:
This was talked about with @cloudhead, since we don't require any additional information at this point, and we can check for an OK response for success.
Any reason what benefit we would have with the `success: true` body?

Reply:
returning empty response would mean any empty response in the future / other routes can be counted as a success as well, which might not hold true, e.g. if someone else implements it and is not aware of this.

we don't lose much by returning a `success: true`, it's just standard.

Reply:
Ok makes sense, will add it 👍  thanks!

Reply:
Done in c9834bd